index.html

```diff
@@ -26,27 +26,10 @@
         </nav>
```

Comment:
Asiu, co myślisz o tym, żeby to dodać do komponentu game mode?

Reply:
@lilareliga jestem za - mam jeszcze chwilkę, to może wrzucę w ramach tego pra?

Reply:
poprawione